### ISSUE TITLE
shell.nix should suggest adding host_platform to all (more?) commands, not just test

### ISSUE THREAD

**Issue by ulysses4ever** [tweag/rules_haskell#939]
**Is your feature request related to a problem? Please describe.**
shell.nix [suggests](https://github.com/tweag/rules_haskell/blob/6e4bcdb877dcf3e958d1bcca915426177fd387e2/shell.nix#L44-L46) adding `.bazelrc.local` with the following:
```
test --host_platform=@io_tweag_rules_haskell//haskell/platforms:''${ARCH}_x86_64_nixpkgs
```
But `host_platform` is needed to other commands, it seems. E.g. to build `tutorial`, which fails without it.

**Describe the solution you'd like**
shell.nix's suggestion should extend to more commands. `common` might be the right replacement for `test`.

**Comment by Profpatsch** [comment]
We have a symlink of `.bazelrc(.local)` from both the `tutorial` and `example` subfolders, linking back to the ones in the project root. So changing it in the project root automatically copies over to these sub-workspaces.

**Comment by mboes** [comment]
Closed  by #939, as the commit message in that message says. Issue auto-close doesn't seem to be working anymore. Could this be related to our use of Mergify?
